2.cellprofiler_analysis/NF1_analysis_plate1_plate2.cppipe

```diff
@@ -10,14 +10,14 @@ Images:[module_num:1|svn_version:'Unknown'|variable_revision_number:2|show_windo
     Filter images?:Images only
     Select the rule criteria:and (extension does isimage) (directory doesnot containregexp "[\\\\/]\\.")
 
-Metadata:[module_num:2|svn_version:'Unknown'|variable_revision_number:6|show_window:False|notes:['The Metadata module optionally allows you to extract information describing your images (i.e, metadata) which will be stored along with your measurements. This information can be contained in the file name and/or location, or in an external file.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+Metadata:[module_num:2|svn_version:'Unknown'|variable_revision_number:6|show_window:False|notes:['Had to change the metadata regular expression to remove plate.The value it was associating as plate in the file name is arbitary and the same between all plates in the NF1 project. ', '', 'Plate is not extracted from the metadata so across-all-plate feature processing can not occur.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
```

Comment:
Thank you! I am only able to the notes to the file from the GUI, the rest of specified from CellProfiler itself so I think it would be a bad idea to change.